.github/workflows/generate-android-vectors.yml

```diff
@@ -0,0 +1,34 @@
+name: Generate android vectors
+on:
+  push:
+    branches:
+      master
```

Comment:
Den her er jeg usikker på, vet ikke hva som er bedst å gjøre (tenker @skjalgepalg har bedre kontroll på hva som har vært vanlig å gjøre), men tenker det er fint å starte med dette og så får vi se hva som vi gjør for fremtiden 😄 

Tenker uansett det er lurt å slenge på `workflow_dispatch:` slik det er mulig å starte det manuelt fra en branch :)

Reply:
Tok egentlig bare utgangspunkt i sånn det er gjort i [snowplow-native](https://github.com/nrkno/snowplow-native/blob/main/.github/workflows/publish-kotlin.yml), tror det er sånn det er gjort i de fleste andre bibliotekene våre

Reply:
Det kan bare være veldig bra for å teste om endringer fungerer :) Men lurer da på om vi ikke bør release 🤔 Det kan vi evt. gjøre noe ved senere - så lenge den blir slengt på før merge er det lettere å endre etterhvert :)

Reply:
Litt sent med innspill kanskje, men tenker det kan være lurt med `workflow_dispatch` som @Anigif nevner. for eksempel vil alle små devdependency bumps fra dependabot gi en push på master uten at vi nødvendigvis behøver ny versjon av vektorene. Det går kanskje an å kjøre ved oppdatert versjon i repo? Men enig i at vi kan se på dette på sikt.

Reply:
`workflow_dispatch` fikk vi lagt til :) Hvordan er deres release flow for vanlige releases? Lurer litt på om man kan streamline med de?